docs-src/concepts/what-is-tctl-v1.md

```diff
@@ -8,7 +8,11 @@ tags:
   - tctl
 ---
 
-This documentation reflects tctl version 1.16
+:::note
+
+This documentation reflects tctl version 1.17
```

Comment:
@DJSanti - 
Looks like we should also change the other v1.16 to v1.17 
Can you do a quick search and replace in this PR?